com.github.Eloston.ungoogled-chromium.yaml

```diff
@@ -0,0 +1,199 @@
+app-id: com.github.Eloston.ungoogled-chromium
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: chromium
+separate-locales: false
+finish-args:
+  - '--require-version=1.8.2'
+  - '--device=all'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=pulseaudio'
+  - '--socket=x11'
+  - '--env=GTK_PATH=/app/lib/gtkmodules'
+  - '--socket=cups'
+  - '--talk-name=org.freedesktop.FileManager1'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.freedesktop.secrets'
+  - '--talk-name=org.gnome.SessionManager'
+  - '--own-name=org.mpris.MediaPlayer2.chromium.*'
+  - '--system-talk-name=org.freedesktop.Avahi'
+  - '--system-talk-name=org.freedesktop.GeoClue2'
+  - '--system-talk-name=org.freedesktop.UPower'
+  - '--filesystem=host-etc'
```

Comment:
Why not remove `host-etc`?

Reply:
Chromium policy and flags are in `/etc/chromium`. Also Native Messaging Hosts